Dockerfile

```diff
@@ -8,18 +8,18 @@ ARG GITHUB_SHA
 ENV CGO_ENABLED=0 GOOS=linux GO111MODULE=on
 
 WORKDIR /go/src/github.com/virtual-vgo/vvgo
-COPY go.mod go.sum .
+COPY go.mod go.sum ./
 RUN go mod download
 
 COPY . .
-RUN --mount=type=cache,target=/go/pkg/mod \
+RUN \
+	# BEGIN BUILDKIT
+    --mount=type=cache,target=/go/pkg/mod \
     --mount=type=cache,target=/root/.cache/go-build \
+	# END BUILDKIT
     go generate \
     && go build -v -o /vvgo
 
-FROM builder as tester
-CMD ["go", "test", "-v", "./..."]
```

Comment:
Lets leave the test commands in. These are unit tests, so they are pretty cheap to run and it gives us validation that the fly.io docker builder isn't doing something weird.

Reply:
Fair enough.